### DISCUSSION TITLE
Using <fieldset> within <fildset>

### DISCUSSION THREAD

**Discussion by sylviefaulkner** [wet-boew/wet-boew#9279] [Q&A] [answered]
I have research the web if &lt;fieldset&gt; within &lt;fildset&gt; and &lt;heading&gt; within &lt;legend&gt; could be used in html5
I found information on the California Community Colleges Accessibility Center website that this was acceptable in html5.
I tested a demo that I created using NVDA and it works like I expect it to work but my form looks terrible.
My guess is that the styling for fieldset fieldset don't exist in the theme.min.css so I was wondering if there is a plan to add this feature in the WET templates. The reason I'm asking is because the form that we are working with is complex and has multiple section where  each section are identical. Visually, it is easy to see the heading above each section but when we generate the error message, many error are the same text so when you click on the error message to take you to the location, the heading is not read if it is not grouped. Then I thought using the aria-describedby to address this problem but then the screen reader repeats the title at every label so not very user friendly. 
Here is a simplified sampling of code I created to display the type of coding that I'm looking for:
&lt;section&gt;<br>
&lt;fieldset&gt;<br>
&lt;legend&gt;&lt;h2&gt;Animal for companion (optional)&lt;/h2&gt;&lt;/legend&gt;<br>
&lt;fieldset class=&quot;chkbxrdio-grp&quot;&gt;<br>
&lt;legend&gt;&lt;span class=&quot;field-name&quot;&gt;Rat&lt;/span&gt;&lt;/legend&gt;<br>
&lt;div class=&quot;checkbox&quot;&gt;<br>
&lt;label for=&quot;Rat1&quot;&gt;<br>
&lt;input type=&quot;checkbox&quot; name=&quot;Rat&quot; value=&quot;Rat1&quot; id=&quot;Rat1&quot; /&gt;White&lt;/label&gt;<br>
&lt;/div&gt;<br>
&lt;/fieldset&gt;<br>
&lt;fieldset class=&quot;chkbxrdio-grp&quot;&gt;<br>
&lt;legend&gt;&lt;span class=&quot;field-name&quot;&gt;Gerbil&lt;/span&gt;&lt;/legend&gt;<br>
&lt;div class=&quot;checkbox&quot;&gt;<br>
&lt;label for=&quot;Gerbil2&quot;&gt;<br>
&lt;input type=&quot;checkbox&quot; name=&quot;Gerbil&quot; value=&quot;Gerbil2&quot; id=&quot;Gerbil2&quot; /&gt;Brown&lt;/label&gt;<br>
&lt;/div&gt;<br>
&lt;/fieldset&gt;<br>
&lt;fieldset class=&quot;chkbxrdio-grp&quot;&gt;<br>
&lt;legend&gt;&lt;span class=&quot;field-name&quot;&gt;Guinea pig&lt;/span&gt;&lt;/legend&gt;<br>
&lt;div class=&quot;checkbox&quot;&gt;<br>
&lt;label for=&quot;Guinea1&quot;&gt;<br>
&lt;input type=&quot;checkbox&quot; name=&quot;Guinea&quot; value=&quot;Guinea1&quot; id=&quot;Guinea1&quot; /&gt;Long hair&lt;/label&gt;<br>
&lt;/div&gt;<br>
&lt;div class=&quot;checkbox&quot;&gt;<br>
&lt;label for=&quot;Guinea2&quot;&gt;<br>
&lt;input type=&quot;checkbox&quot; name=&quot;Guinea&quot; value=&quot;Guinea2&quot; id=&quot;Guinea2&quot; /&gt;Short hair&lt;/label&gt;<br>
&lt;/div&gt;<br>
&lt;/fieldset&gt;<br>
&lt;fieldset class=&quot;chkbxrdio-grp&quot;&gt;<br>
&lt;legend&gt;&lt;span class=&quot;field-name&quot;&gt;Cat&lt;/span&gt;&lt;/legend&gt;<br>
&lt;div class=&quot;checkbox&quot;&gt;<br>
&lt;label for=&quot;Cat1&quot;&gt;<br>
&lt;input type=&quot;checkbox&quot; name=&quot;Cat&quot; value=&quot;Cat1&quot; id=&quot;Cat1&quot; /&gt;Long hair&lt;/label&gt;<br>
&lt;/div&gt; <br>
&lt;div class=&quot;checkbox&quot;&gt;<br>
&lt;label for=&quot;Cat2&quot;&gt;<br>
&lt;input type=&quot;checkbox&quot; name=&quot;Cat&quot; value=&quot;Cat2&quot; id=&quot;Cat2&quot; /&gt;Short hair&lt;/label&gt;<br>
&lt;/div&gt; <br>
&lt;/fieldset&gt;<br>
&lt;fieldset class=&quot;chkbxrdio-grp&quot;&gt;<br>
&lt;legend&gt;&lt;span class=&quot;field-name&quot;&gt;Dog&lt;/span&gt; &lt;/legend&gt;<br>
&lt;div class=&quot;checkbox&quot;&gt;<br>
&lt;label for=&quot;Dog1&quot;&gt;<br>
&lt;input type=&quot;checkbox&quot; name=&quot;Dog&quot; value=&quot;Dog1&quot; id=&quot;Dog1&quot; /&gt;Long hair&lt;/label&gt;<br>
&lt;/div&gt;<br>
&lt;div class=&quot;checkbox&quot;&gt;<br>
&lt;label for=&quot;Dog2&quot;&gt;<br>
&lt;input type=&quot;checkbox&quot; name=&quot;Dog&quot; value=&quot;Dog2&quot; id=&quot;Dog2&quot; /&gt;Short hair&lt;/label&gt;<br>
&lt;/div&gt;<br>
&lt;div class=&quot;checkbox&quot;&gt;<br>
&lt;label for=&quot;Dog3&quot;&gt;<br>
&lt;input type=&quot;checkbox&quot; name=&quot;Dog&quot; value=&quot;Dog3&quot; id=&quot;Dog3&quot; /&gt;Non-shedding&lt;/label&gt;<br>
&lt;/div&gt;<br>
&lt;/fieldset&gt;<br>
&lt;/fieldset&gt;<br>
&lt;/section&gt;

Thank you

**Comment by ghost** [comment, marked as answer]
This be more suited as an issue, but your issue is related to Bootstrap 3 (https://getbootstrap.com/docs/3.4/).

```
fieldset {
  min-width: 0;
  padding: 0;
  margin: 0;
  border: 0;
}
```

I believe the `min-width: 0;` set by bootstrap is causing the issues, maybe overriding this we'll fix this.
